src/ncar-bufr2nc-fortran/define_mod.f90

```diff
@@ -19,8 +19,7 @@ module define_mod
 integer(i_kind), parameter :: nvar_met          = 6
 integer(i_kind), parameter :: nvar_info         = 9  ! number of metadata
 integer(i_kind), parameter :: nsen_info         = 7  ! number of sensor metadata
-integer(i_kind), parameter :: ninst             = 12
-!integer(i_kind), parameter :: ninst             = 13 ! including airs
```

Comment:
was not aware you were looking at these routines as well. @rmclaren thank you for trying to keep these in sync were these tested?  There was an independent fork of these decoders but would be nice to make this set the standard ones

Reply:
These changes were unintentional, I',m not sure how they happened. I went ahead and reverted this change and other changes like this.